hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/opt/physical/InsertPhysicalRel.java

```diff
@@ -38,18 +40,14 @@
             RelOptTable table,
             Prepare.CatalogReader catalogReader,
             RelNode input,
-            Operation operation,
-            List<String> updateColumnList,
-            List<RexNode> sourceExpressionList,
             boolean flattened
     ) {
-        super(cluster, traitSet, table, catalogReader, input, operation,
-                updateColumnList, sourceExpressionList, flattened);
+        super(cluster, traitSet, table, catalogReader, input, INSERT, null, null, flattened);
     }
 
     @Override
     public PlanNodeSchema schema(QueryParameterMetadata parameterMetadata) {
-        return OptUtils.schema(getTable());
+        return new PlanNodeSchema(singletonList(QueryDataType.BIGINT));
```

Comment:
I tried this here and the test passed. The DML operations don't return any rows, so this method is likely unused.
```suggestion
        throw new UnsupportedOperationException();
```

Reply:
> it isn't used yet though
no it's not